modules/claims_api/app/controllers/claims_api/v1/application_controller.rb

```diff
@@ -29,7 +29,8 @@ def validate_veteran_identifiers(require_birls: false) # rubocop:disable Metrics
 
         if require_birls && target_veteran.participant_id.present? && target_veteran.birls_id.blank?
           raise ::Common::Exceptions::UnprocessableEntity.new(detail:
-            'Unable to locate Veteran BIRLS ID. Please contact the Digital Transformation Center (DTC) at 202-921-0911 for assistance.') # rubocop:disable Layout/LineLength
+            'Unable to locate Veteran BIRLS ID. '\
```

Comment:
realized there's a couple other places where ignoring the `LineLength` rubocop rule is easily remedied